client/__tests__/e2e/e2e.test.ts

```diff
@@ -165,12 +157,6 @@ for (const testDataset of testDatasets) {
         for (const label of Object.keys(
           data.categorical
         ) as (keyof typeof data.categorical)[]) {
-          const element = await page
-            .getByTestId(`category-${label}`)
-            .innerHTML();
-
-          expect(element).toMatchSnapshot();
```

Comment:
removed this one and the following, since the bottom chromatic snapshot should capture them anyway